app/javascript/gobierto_data/webapp/components/Dataset.vue

```diff
@@ -252,7 +254,8 @@ export default {
       labelQueries: I18n.t("gobierto_data.projects.queries") || "",
       labelVisualizations:
         I18n.t("gobierto_data.projects.visualizations") || "",
-      labelDownload: I18n.t("gobierto_data.projects.download") || ""
+      labelDownload: I18n.t("gobierto_data.projects.download") || "",
+      registrationDisabled: this.$root.$data.registrationDisabled === "true" ? true : false
```

Comment:
Conditional expressions are evaluated as Boolean. So, you won't need the ternary.

```suggestion
      registrationDisabled: this.$root.$data.registrationDisabled === "true"
```